src/Common/DigipolisPropertiesAwareInterface.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
Using the trait is a way to implement the interface. You can implement the interface without using the trait. 

It's a way for us to determine if a command is 'DigipolisPropertiesAware'.